### PR TITLE
Add .circleci/config.yml

### PR DESCRIPTION
**Pull request by AniaMakes**
PR to add CircleCI to help with key maintenance. 

https://financialtimes.atlassian.net/secure/RapidBoard.jspa?rapidView=1137&projectKey=NOPS&modal=detail&selectedIssue=NOPS-115

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+version: 2.1
+orbs:
```
Comment (binaryberry) [inline .circleci/config.yml]: Most of our repos seem to be using docker images for their CI runs, see: https://github.com/Financial-Times/next-article/blob/master/.circleci/config.yml#L4

I'm not sure of the reasons behind using that (Kara might know more!) but would be best to be consistent with the rest of the FT.com estate if possible.

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
 orbs:
   node: circleci/node@1.1.6
+  dotenv: anilanar/dotenv@volatile
 jobs:
   build-and-test:
     executor:
       name: node/default
     steps:
       - checkout
+      - dotenv/source:
```
Comment (binaryberry) [inline .circleci/config.yml]: For a cleaner github history it'd be good to remove the commits about adding and removing the dotenv orb mention - a github rebase -interactive should help do that.

**Review by binaryberry — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (apaleslimghost) [reply .circleci/config.yml]: i think the only reasoning behind Docker was that we migrated to CircleCI 2 before Orbs were available. however, it's good to keep things consistent

**Review by binaryberry — APPROVED** [review]
Looks good! My [comment](https://github.com/Financial-Times/alphaville-es-interface-service/pull/16#discussion_r414563145) about the Github history stands but I don't feel strongly about it.

**Pull request merged**